docs/orchestrator.md

```diff
@@ -298,6 +244,14 @@ Now, you can restart the orchestrator, and it should start signing.
 
 Note: A validator set change is triggered if more than 5% of the total staking power of the network changes. This means that even if you change your EVM address, and you don't see your orchestrator signing, it's alright. Just wait until the validator set changes, and then your orchestrator will automatically start signing.
 
+### Updating the EVM address if its corresponding private key is lost
+
+If, for some reason, the private key, corresponding to the EVM account that was registered above, was lost, or some validator wanted to change it for some reason, then they can re-register another EVM address for their validator using the same above command using the new EVM address and it should work.
+
+If the validator still has access to the previous running orchestrator, it would be safer to keep it running in a separate process, and initializing a new orchestrator in a new home directory, and running it using the new EVM address. Then, once the new orchestrator starts signing, the old one can be stopped.
```

Comment:
```suggestion
If the validator still has access to the previously running orchestrator, it would be safer to keep it running in a separate process, initialize a new orchestrator in a new home directory, and run it using the new EVM address. Then, once the new orchestrator starts signing, the old one can be stopped.
```